content/en/apps/examples/anc.md

```diff
@@ -1,16 +1,55 @@
+{{< see-also page="apps/concepts" title="Concepts that will help you understand how CHT applications are built" >}}
 
-[![ANC Overview](preview-anc-ref-app-overview.png)](anc-reference-app-overview.pdf)
+## Problem Being Addressed
 
-If you want to modify our ANC Reference Application for your project or context, read our [configuration best practices]({{< ref "design/apps" >}}) or learn how to [develop your own community health apps]({{< ref "apps" >}}).
+Maternal health is the cornerstone of many community health programs. Pregnancy can be a [vulnerable time](https://www.who.int/health-topics/maternal-health) for women living in communities far from medical care. Additionally, the short window immediately after pregnancy is a critical time for catching life-threatening danger signs for the new mother and baby.
+
+## Solution Overview
+
+The ANC workflow in this reference app helps ensure that women are safe and supported in a timely manner throughout their pregnancy. Early and timely antenatal care visits and increased coordination between CHWs and clinics lead to an increase in facility-based deliveries, which in turn has a strong correlation to maternal and newborn survival. CHT deployments of ANC workflows are used by health workers and clinical staff to:
+
+- Provide on-time antenatal care visits
+- Educate at each stage of pregnancy
+- Screen and report danger signs
+- Refer and encourage facility delivery
+- Improve delivery rates in facilities with skilled care
+
+## Forms Hierarchy
+
+Once a program [hierarchy]({{< relref "apps/concepts/hierarchy" >}}) of people and places is established, forms are added at different levels. This diagram indicates the forms that can be filled about a person in the app (in this case, family members at the household level), as well as the person/user who will access these forms and make the reports (CHWs at the CHW Area level). Some forms are accessible as actions from the family member’s profile as actions, others from the CHW’s task list as tasks, and some as either. 
+
+![forms-hierarchy.png](forms-hierarchy.png)
+
+{{< see-also page="apps/reference/forms/app" title="Controlling form properties" >}}
+
+## Pregnancy and Delivery Workflows
+
+Next, pregnancy and delivery care workflows are defined to connect form actions and data with people. Detailed documentation to ANC Reference App forms and task schedules are linked from the workflow diagrams below. Accompanying this documentation are tips and insights into the design decisions made along the way, and suggestions for how and where to customize the forms.
+
+[![Pregnancy Workflow](preview-pregnancy-workflow.png)](pregnancy-workflow.pdf)
+
+<br /> 
+
+[![Delivery Workflow](preview-delivery-workflow.png)](delivery-workflow.pdf)
```

Comment:
Given that we are relying on the linked PDF, it is time to convert these to markdown/HTML.

@MaxDiz, can you attempt to do it first with markdown tables? If that doesn't have the desired effect we can then more easily adapt the markdown to a HTML layout, which I can help with.

Reply:
I messed around with converting to a markdown table yesterday. It unfortunately obscures that it's a workflow (ie it's not clear whether to read down or across). Also not sure how to include the icons which add important context.

What do you think about doing an interim merge with the pdf's, and then opening a new issue to create an html solution?

Reply:
This isn't a big lift, and I'd prefer we take it on now rather than introducing PDFs that we rely on for links.

Reply:
@MaxDiz do you have the text markdown version of this? Once you commit that I can then HTML-ify it.